packages/palette/src/elements/Modal/Modal.tsx

```diff
@@ -230,8 +230,10 @@ const ModalElement = styled(AnimatedView)<TransitionElementProps>`
 const ModalScrollContent = styled(Box)<ModalScrollContentProps>`
   height: 100%;
   width: 100%;
-  overflow: scroll;
-  padding-bottom: ${props => (!!props.FixedButton ? "112px" : space(2) + "px")};
+  max-height: ${props =>
+    !!props.FixedButton ? "calc(100vh - 172px)" : "calc(100vh - 80px)"};
```

Comment:
this is so so minor, but `Boolean(props.FixedButton)` is more readable at a glance

Reply:
fair, but `!!` is also ubiquitous.